docs/userguide.md

```diff
@@ -299,9 +302,11 @@ kubectl delete federatednamespaceplacement test-namespace
 
 ### Deployment Cleanup
 
-Run the following command to perform a cleanup of the cluster registry and
-federation deployments:
+Assume you are runnig the deloyment cleanup in cluster1. Run the following command to perform
```

Comment:
Same comment as above. I'd reword this. s/runnig/running/

Reply:
I deleted this assume, as the pre-condition already mean we are using `cluster 1`